iati/core/default.py

```diff
@@ -102,6 +105,73 @@ def codelists(version=None, use_cache=False):
     return _CODELISTS
 
 
+def codelist_mapping(version=None):
+    """Define the mapping process which states where in a Dataset you should find values on a given Codelist.
+
+    version (str): The version of the Standard to return the data files for. Defaults to None. This means that the mapping file is returned for the latest version of the Standard.
+
+    Returns:
+        dict of dict: A dictionary containing mapping information. Keys in the first dictionary are Codelist names. Keys in the second dictionary are `xpath` and `condition`. The condition is `None` if there is no condition.
+
+    Todo:
+        Test this.
+        Stop filtering out filters for org files.
+
+    """
+    path = iati.core.resources.get_codelist_mapping_path()
+    mapping_tree = iati.core.resources.load_as_tree(path)
+    mappings = defaultdict(list)
+
+    for mapping in mapping_tree.getroot().xpath('//mapping'):
+        codelist_name = mapping.find('codelist').attrib['ref']
+        codelist_location = mapping.find('path').text
+        if 'organisation' in codelist_location:
+            continue
+        try:
+            condition = mapping.find('condition').text
+        except AttributeError:  # there is no condition
+            condition = None
+
+        mappings[codelist_name].append({
+            'xpath': codelist_location,
+            'condition': condition
+        })
+
+    return mappings
+
+
+def ruleset(version=None):
+    """Locate the default Ruleset for the specified version of the Standard.
+
+    Args:
+        version (str): The version of the Standard to return the Ruleset for. Defaults to None. This means that the latest version of the Ruleset is returned.
+
+    Returns:
+        iati.core.ruleset.Ruleset: The default Ruleset for the specified version of the Standard.
+
+    Raises:
+        ValueError: When a specified version is not a valid version of the IATI Standard.
+
+    Todo:
+        Actually handle versions, including errors.
+
+    """
+    name = 'standard_ruleset'
+
+    path = iati.core.resources.get_ruleset_path(name, version)
+    ruleset_str = iati.core.resources.load_as_string(path)
+
+    return iati.core.rulesets.Ruleset(ruleset_str)
+
+def ruleset_schema(version=None):
+    name = 'ruleset_schema'
```

Comment:
As with the `standard_ruleset`, this should be defined as a constant in `resources.py`.

Reply:
Noted in https://github.com/IATI/iati.core/pull/49#issuecomment-327143653

Reply:
done